src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1575,6 +1575,9 @@ function ($a) {
                                         ->defaultNull()
                                         ->info('Transport name to send failed messages to (after all retries have failed).')
                                     ->end()
+                                    ->booleanNode('include_stack_trace_in_error')
```

Comment:
I'm not too crazy about the name of the option, maybe someone has a better suggestion.